README.md

```diff
@@ -289,14 +265,12 @@ graph TD;
 > P.S. - на сервере стоит ось `FreeBSD`
 
 <a id="Подключение"></a>
-<details open>
-<summary align="center"><h3><b> Подключение и загрузка файлов </b></h3></summary>
-
-<a id="авторизация"></a>
+<details>
+<summary align="center"><h2><b> Подключение и загрузка файлов </b></h2></summary>
 
```

Comment:
you removed "open" tag for <details open>

because of that, you will be unable to locate to ANY inner header of that details

it would be possible ONLY if the <details> would be opened. So i suggest you opening it. You can test it locally by clicking on the table of contents and see that you can't navigate from it inside "Подключение" when it's closed